src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_Dependencies.cs

```diff
@@ -0,0 +1,91 @@
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Maestro.ScenarioTests.ObjectHelpers;
+using Microsoft.DotNet.Maestro.Client.Models;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace Maestro.ScenarioTests
+{
+    [TestFixture]
+    public class ScenarioTests_Dependencies : MaestroScenarioTestBase
+    {
+        private readonly IImmutableList<AssetData> source1Assets = ImmutableList<AssetData>.Empty;
+        private readonly IImmutableList<AssetData> source2Assets = ImmutableList<AssetData>.Empty;
+        private readonly IImmutableList<AssetData> targetAssets = ImmutableList<AssetData>.Empty;
+        private string source1RepoUri;
```

Comment:
Same comments as before. It seems like all these fields should just be local variables to the test... they only apply to a single test.